internal/integration/api/discovery.go

```diff
@@ -66,18 +66,18 @@ func (suite *DiscoverySuite) TearDownTest() {
 //
 //nolint:gocyclo
 func (suite *DiscoverySuite) TestMembers() {
-	nodes := suite.DiscoverNodes(suite.ctx)
+	nodes := suite.DiscoverNodeInternalIPs(suite.ctx)
 	expectedTalosVersion := fmt.Sprintf("Talos (%s)", suite.Version)
 
-	for _, node := range nodes.Nodes() {
+	for _, node := range nodes {
```

Comment:
we could have used here the full list of IPs and match as a subset of member addresses